lib/dynflow.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
+
 require 'apipie-params'
 require 'algebrick'
-require 'thread'
```

Comment:
Just to make sure, this isn't being used, is it?

Reply:
We're using threads, but it seems to be auto-required. Or to put it the other way around, we don't seem to be using anything related to threads that we'd have to explicitly require